src/commands/build.ts

```diff
@@ -40,18 +41,29 @@ export class BuildCommand implements Command {
       description: 'Flatten dependency tree downloads by inserting ' +
         'additional `<link rel="prefetch">` tags into ' +
         'entrypoint and `<link rel="import">` tags into fragments and shell.'
+    },
+    {
+      name: 'html.collapseWhitespace',
+      type: Boolean,
+      description: 'Collapse whitespace in HTML files'
     }
   ];
 
   run(options, config): Promise<any> {
     // Defer dependency loading until this specific command is run
     const build = require('../build/build').build;
 
-    let buildOptions = {
+    let buildOptions: BuildOptions = {
       sources: options.sources,
       swPrecacheConfig: options['sw-precache-config'],
-      insertDependencyLinks: options['insert-dependency-links']
+      insertDependencyLinks: options['insert-dependency-links'],
+      html: {},
```

Comment:
They are not required, I just didn't want to make them conditionally based on the flags.